### PR TITLE
Refresh UI

### PR DESCRIPTION
**Pull request by timja**
Unblocks https://github.com/jenkinsci/jenkins/pull/7717
Test issue is in https://github.com/jenkinsci/acceptance-test-harness/pull/1189

Changes:
- Removed icons from headings. see guidance at: https://weekly.ci.jenkins.io/design-library/Symbols/
- Used built-in controls rather than custom html. This is what is blocking the core PR as it doesn't adapt to new markup in the controls.
- Incrementalise the plugin to make testing PRs across components simple
- Refreshed pom to latest standards from the [archetypes](https://github.com/jenkinsci/archetypes)
- Changed legacy tango icons to [symbols](https://weekly.ci.jenkins.io/design-library/Symbols/). <- Not done on the run/edit scripts page but done everywhere else I think
- Removed no longer needed tables to divs compatibility (p:blockWrapper)
- Changed Submit to Save in a few places
- Put the Scripter management link in the configuration category so it no longer shows up in 'uncategorized'

image

image

### Testing done

Clicked through UI and saved and edited scripts

### Submitter checklist

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/scriptSettings.properties [hunk 1]
```diff
@@ -29,7 +29,7 @@ uploadtext=Select a Groovy script from your local system to be uploaded (*.groov
 UploadEncoding=Uploaded files should be encoded with UTF-8.
 Permission = Allow usage in build step
 Restriction = Restriction
-RestrictionDescription = Script is always executed on Master
+RestrictionDescription = Script is always executed on the controllerMaster
```
Comment (mtughan) [inline src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/scriptSettings.properties:32]: I think you meant to get rid of the term "master" here?
```suggestion
RestrictionDescription = Script is always executed on the controller
```

**Review by mtughan — CHANGES_REQUESTED** [review]
Looks generally fine, but I will do some testing with these changes before merging it. There's also the one comment I have below.

File: pom.xml [hunk 2]
```diff
@@ -3,27 +3,15 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>4.40</version>
+		<version>4.64</version>
```
Comment (timja) [inline pom.xml:6]:
```suggestion
		<version>4.65</version>
```

File: Jenkinsfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
-buildPlugin(configurations: [
-    [platform: 'linux', jdk: '8'],
+buildPlugin(useContainerAgent: true,
+  configurations: [
     [platform: 'linux', jdk: '11'],
-    [platform: 'windows', jdk: '11']
+    [platform: 'linux', jdk: '17'],
```
Comment (timja) [inline Jenkinsfile:4]:
```suggestion
    [platform: 'linux', jdk: '17'],
    [platform: 'windows', jdk: '11']
```

**Comment by timja** [comment]
(I've re-enabled Windows builds, turned out to be a general regression affecting a few repositories and has been fixed in the test harness)

**Review by mtughan — DISMISSED** [review]
Thanks @timja, changes look good from my side then. Will try to test this in the next few days and merge if all looks good.

**Comment by mtughan** [comment]
@timja, I rebased this PR as it conflicted with #60. Can you double-check that all looks good from your side with it?

**Comment by timja** [comment]
> @timja, I rebased this PR as it conflicted with #60. Can you double-check that all looks good from your side with it?

Merge LGTM

**Comment by timja** [comment]
@mtughan anything you are waiting on here?

**Review by mtughan — APPROVED** [review]
@timja, just on me to have time to review it. I just finished up a task I've been working on for the past 2 weeks, so I'm going to work on cleaning up a bunch of outstanding tasks on the plugin.

**Pull request merged**